README.md

```diff
@@ -1 +1,112 @@
-# planner
+# Planer - University Scheduler
+
+Welcome to the repository of the Solvro project, a student organization at the Wrocław University of Science and Technology!
+
+[![Welcome Page](https://i.imgur.com/PSnVCNN.png)](https://i.imgur.com)
+
+## Project Goal
+
+The goal of the Solvro Planner project is to create an intuitive and user-friendly application that helps students plan their academic schedule. The planner is designed to minimize the time spent manually adjusting the timetable while giving full control over the schedule.
+
+## Current Team
+
+- @Szymczek - Project Manager
+- @Rei-x - Techlead
+- @unewMe - Fullstack Developer
+- @D0dii - Fullstack Developer
+- @olekszczepanowski - Fullstack Developer
+
+## Technologies
+
+- Next.js
+- React.js
+- Tailwind
+- Typescript
+- Colify
+- Directus CMS
+
+## Links
+- https://planer.solvro.pl
+- https://solvro.pwr.edu.pl
+
+## Statistics
+
+Statistics for Solvro Planner version 1.0
+
+[![Statistics](https://i.imgur.com/My4U8lY.png)](https://i.imgur.com)
```

Comment:
możesz też podlinkować statystyki gdzies tutaj i to nie są "Statistics" tylko "Analitics"